### PR TITLE
Make `accepted_version_id` optional throughout SDK

### PR DESCRIPTION
**Pull request by shannynalayna**
This updates the `accepted_version_id` field in the purchase order object to be optional throughout the SDK. This previously was defined as a required string, however this value may not always be defined. A purchase order version must first be validated that it is able to be "accepted". Therefore, this value should be optional to allow for the cases where a purchase order does not have an accepted version (yet).

### CODE REVIEW

File: contracts/purchase_order/src/workflow.rs [hunk 1]
```diff
@@ -398,3 +400,12 @@ fn collaborative_sub_workflow() -> SubWorkflow {
         .add_starting_state("create")
         .build()
 }
+
+impl fmt::Display for POWorkflow {
```
Comment (peterschwarz) [inline contracts/purchase_order/src/workflow.rs:404]: This seems like an unrelated change

Comment (shannynalayna) [reply contracts/purchase_order/src/workflow.rs]: Removed

**Review by peterschwarz — APPROVED** [review]

**Review by dnewh — APPROVED** [review]
